lib/modules/manager/gomod/artifacts.ts

```diff
@@ -113,14 +116,23 @@ function addAuthFromHostRule(
 
 function getUpdateImportPathCmds(
   updatedDeps: PackageDependency[],
-  { constraints, newMajor }: UpdateArtifactsConfig
+  { constraints }: UpdateArtifactsConfig
 ): string[] {
   const updateImportCommands = updatedDeps
-    .map((dep) => dep.depName!)
-    .filter((x) => !x.startsWith('gopkg.in'))
+    .map(({ depName, newVersion }) => ({
+      depName: depName!,
+      newMajor: major(newVersion!),
+    }))
+    // Skip path upates going from v0 to v1
+    .filter(
+      ({ depName, newMajor }) => !depName.startsWith('gopkg.in') && newMajor > 1
```

Comment:
The comment says skip updates from v0 to v1, but this it any update which goes to >1?

Reply:
I think I just got confused by adding gopkg.in to this logic. We skip if either it's from gopkg.in (any) or if newMajor goes from 0 to 1?

Reply:
Yes, this will skip if its either from gopkg.in or if the upgrade is v0->v1